src/WeakMap.php

```diff
@@ -86,6 +86,7 @@ public function offsetExists($object) : bool
             return false;
         }
 
+        #[\ReturnTypeWillChange]
```

Comment:
What's the point of this one, as the code will never be executed on PHP 8.1+?

Reply:
it is not needed